src/services/document-service.spec.js

```diff
@@ -972,7 +972,7 @@ describe('document-service', () => {
         description: 'Description 1',
         slug: 'doc-1',
         sections: [],
-        tags: ['music', 'instructor', 'Dj.D', 'Cretu'],
+        tags: ['music', 'instructor', 'Dj.D', 'Cretu', '1'],
```

Comment:
I'm sure of the author of this "Cretu" name in code 😊

Reply:
It was **not** Maria Magdalena :rofl:

Reply:
😂